docs/sql-programming-guide.md

```diff
@@ -137,8 +137,15 @@ df.show()
 
 DataFrames provide a domain-specific language for structured data manipulation in [Scala](api/scala/index.html#org.apache.spark.sql.DataFrame), [Java](api/java/index.html?org/apache/spark/sql/DataFrame.html), and [Python](api/python/pyspark.sql.html#pyspark.sql.DataFrame).
 
-Here we include some basic examples of structured data processing using DataFrames:
+<div data-lang="python"  markdown="1">
```

Comment:
I am not sure if this works. You should put it in the Python part of the example below, right before highlight python.